### PR TITLE
sql: add information_schema.(applicable_roles|enabled_roles) views

### LINKED ISSUE #2646: sql: Add information_schema.applicable_roles table
Need to add another "view" to https://www.cockroachlabs.com/docs/dev/information-schema.html.

### LINKED ISSUE #2648: sql: Add information_schema.enabled_roles table
Need to add another "view" to https://www.cockroachlabs.com/docs/dev/information-schema.html.

### PR DESCRIPTION
**Pull request by spencerkimball**
Fixes #2646
Fixes #2648

### CODE REVIEW

**Comment by spencerkimball** [comment]
+cc @mberhault

File: v2.0/information-schema.md [hunk 1]
```diff
@@ -30,6 +30,17 @@ Views | [tables](#tables), [views](#views)| [`SHOW CREATE VIEW`](show-create-vie
 
 The `information_schema` database is comprised of many views representing your cluster's schema, each of which is detailed below.
 
+### applicable_roles
+
+The `applicable_roles` view identifies all roles whose privileges the current user can use. This implies there is a chain
+of role grants from the current user to the role in question. The current user itself is also an applicable role.
```
Comment (mberhault) [inline v2.0/information-schema.md]: the current user isn't actually included in the view. I [pinged](https://www.postgresql.org/message-id/151747128215.1240.1598372623554241748%40wrigleys.postgresql.org) the postgres docs mailing list about this to figure out if it was odd wording or mismatched docs, no answer. I'm taking their docs to mean: "the listed roles are applicable role; the current user is also an applicable role, but is not listed"

Comment (spencerkimball) [reply v2.0/information-schema.md]: Done.

**Comment by jseldess** [comment]
@spencerkimball, since other PRs have been touching this file, you need to resolve some conflicts before merging is possible.

**Pull request merged**